### PR TITLE
Updated 4.5.0 documentation tags

### PR DESCRIPTION
**Pull request by SadanandGowda**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @SadanandGowda,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://922.qa.docs.sunbird.org/4.5.0/ .<br/> **Note:** Only _4.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://922.qa.docs.sunbird.org/4.5.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/build-and-deploy.md [hunk 1]
```diff
@@ -71,48 +73,48 @@ az storage blob upload-batch --destination sunbird_content_azure_storage_contain
...
-|Build/Core/Learner|release-4.4.0_RC11|<https://github.com/project-sunbird/sunbird-lms-service.git>|Handles user and organizations|
-|Build/Core/Lms|release-4.4.0_RC3|<https://github.com/project-sunbird/sunbird-course-service.git>|Handles courses|
-|Build/Core/Nodebb|github_release_tag: release-4.4.0_RC3, nodebb_branch: v1.16.0|<https://github.com/project-sunbird/sunbird-nodebb.git>|Handles the discussion forum|
-|Build/Core/Notification|release-4.4.0_RC5|<https://github.com/project-sunbird/sunbird-notification-service.git>|Handles notifications|
-|Build/Core/Player|release-4.4.0_RC32|<https://github.com/Sunbird-Ed/SunbirdEd-portal.git>|Handles the UI elements|
+|Build/Core/Learner|release-4.5.0_RC10|<https://github.com/project-sunbird/sunbird-lms-service.git>|Handles user and organizations|
+|Build/Core/Lms|release-4.5.0_RC1|<https://github.com/project-sunbird/sunbird-course-service.git>|Handles courses|
+|Build/Core/Nodebb|github_release_tag: release-3.9.0_RC6, nodebb_branch: v1.16.0|<https://github.com/project-sunbird/sunbird-nodebb.git>|Handles the discussion forum|
```
Comment (keshavprasadms) [inline developer-docs/server-installation/build-and-deploy.md:95]: Please check this line

**Comment by amorphous-1** [comment]
Hello @SadanandGowda,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://922.qa.docs.sunbird.org/4.5.0/ .<br/> **Note:** Only _4.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://922.qa.docs.sunbird.org/4.5.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @SadanandGowda,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://922.qa.docs.sunbird.org/4.5.0/ .<br/> **Note:** Only _4.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://922.qa.docs.sunbird.org/4.5.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**